dask/array/tests/test_cupy_core.py

```diff
@@ -105,6 +105,22 @@
     lambda x: np.nanprod(x),
     lambda x: np.nanmin(x),
     lambda x: np.nanmax(x),
+    pytest.param(
+        lambda x: np.angle(x),
+        marks=pytest.mark.skipif(
+            cupy_version < parse_version("11.0.0rc2"),
```

Comment:
It feels a little strange to rely on release candidates when doing version checks. I wonder if a more direct check like `dask.utils.has_keyword(cupy.angle, "deg")` might be more clear.

Reply:
That is fine by me as well, done in https://github.com/dask/dask/pull/9312/commits/1fee85ab61e89ece8a9cb16e2e6d56b3f7ab3b28. Thanks James!